bdb_transaction_cli/cli.py

```diff
@@ -1,15 +1,132 @@
-# -*- coding: utf-8 -*-
-
 import click
+import json
+
+from bigchaindb_common.crypto import generate_key_pair
+from bigchaindb_common.transaction import Transaction, \
+    Condition, Fulfillment, Metadata, Asset, Ed25519Fulfillment
+
+from bdb_transaction_cli.utils import json_argument, json_option, listify
+
+
+@click.group()
+def main():
+    pass
+
+
+@main.command()
+@click.option('--name', required=False, help=(
+    "Print the keys as shell variables,                             eg: "
+    "`export $(bdb generate_keys --name=bob)`"))
+def generate_keys(name):
+    """
+    Generate Ed25519 key pair.
+
+    Generates a random Ed25519 key pair separated by a space character.
+    First value is the public key, second is the private key.
+    """
+    priv, pub = generate_key_pair()
+    fmt = '{pub} {priv}'
+    if name:
+        fmt = '{name}_pub={pub} {name}_priv={priv}'
```

Comment:
In the event that `name` is not given, or `None`, `''`, etc, it will not be echoed which is the public key, and which is the private one. Why is that so?

In other words why the different outputs? Why not simply omit the `name` when not given, but still state which key is which?